run_tests.sh

```diff
@@ -97,6 +101,13 @@ export HOME="/root"
 nitro-cli build-enclave --docker-uri 667861386598.dkr.ecr.us-east-1.amazonaws.com/enclaves-samples:vsock-sample \
 	--output-file test_images/vsock-sample.eif || test_failed
 
+# Build enclave image using Docker ENTRYPOINT instruction
+sed -i 's/CMD/ENTRYPOINT/g' examples/"${ARCH}"/hello/Dockerfile
+
+nitro-cli build-enclave --docker-dir examples/"${ARCH}"/hello --docker-uri hello-entrypoint-usage --output-file test_images/hello-entrypoint-usage.eif || test_failed
```

Comment:
this script does not have exit on error, so we would not know if the command failed. 
I think you would need to do  ||| test_failed.

Reply:
It is currently catching the failure and is looking like this:

nitro-cli build-enclave --docker-dir examples/"${ARCH}"/hello --docker-uri hello-entrypoint-usage --output-file test_images/hello-entrypoint-usage.eif || test_failed